### ISSUE TITLE
[Content styleguide] Content doesn't align with style guidance in VA.gov content style guide. (05.08.3)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#28329]
#### VFS team name
Healthcare Experience

#### VFS product name
Check-in MVP

#### Point of Contact/Reviewers
Allison Christman

---

### Platform Issue
Content doesn't align with style guidance in VA.gov content style guide.

### Issue Details
The time zones are missing from the appointment time

### Supporting Guidance
[Category 5, Issue 8](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

---

### Platform Recommendation
Add time zones to appointment time

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by jparker0425** [comment]
Timezones are currently not included in MVP and the dateTime is pre-formatted from CHIP. This was discussed to look into in phase 2 post-MVP launch. 
https://dsva.slack.com/archives/C022AC2STBM/p1626701890091200

**Comment by shiragoodman** [comment]
adding labels so ticket is consistent with post-pilot process
